backend_test/CMakeLists.txt

```diff
@@ -80,10 +86,15 @@ if(DEFINED SDL_PATH)
 		set(SDL_MAIN SDL2::SDL2main)
 		message(STATUS ${SDL_MAIN} ${IMGUI_SDL_LIBRARY})
 	endif()
-else(DEFINED SDL_PATH)
-	message(STATUS "SDL_PATH not defined")
+else(SDL2_FOUND)
+	if(DEFINED SDL_PATH)
+		message(FATAL_ERROR "Cannot find SDL at SDL_PATH")
+	else(DEFINED SDL_PATH)
+		message(FATAL_ERROR "Cannot find SDL. Maybe try specifying SDL_PATH?")
+	endif(DEFINED SDL_PATH)
+
 	set(IMGUI_SDL_LIBRARY SDL2)
```

Comment:
So this is only set when sdl2 not found?

Reply:
`message(FATAL_ERROR ...)` aborts the build with an error.

If SDL2 is not found, the build fails and suggests to try specifying SDL_PATH
![image](https://user-images.githubusercontent.com/10130860/147393382-9c6d3e52-8516-4bee-9144-1f3d409d015a.png)

If SDL2 is not found at SDL_PATH, the build fails saying it could not find SDL at SDL_PATH :)
![image](https://user-images.githubusercontent.com/10130860/147393398-0e82bd6b-1727-4c6e-b534-9069466c0b20.png)

Reply:
Deleting it will be clearer

Reply:
Well, the original code set this variable unconditionally (even if it failed to find SDL), so I kept it. A quick test on Linux showed that this particular line, `set(IMGUI_SDL_LIBRARY SDL2)`, doesn't really affect the build, at least for me. But I can also move it inside `if(SDL2_FOUND)` block to keep it.
So, what would you prefer? Do I delete this line, or do I move it inside `if(SDL2_FOUND)`?

Reply:
Delete

Reply:
Removed as requested